README.md

```diff
@@ -1,2 +1,227 @@
-# Quantum Network Explorer CLI
-Command Line Interface to interact with the Quantum Network Explorer
+# Quantum Network Explorer ADK
```

Comment:
This bracnh contains the source files in cli instead of adk. I think you should rebase or somthing